Marlin/src/inc/Conditionals_post.h

```diff
@@ -472,7 +472,7 @@
 #elif EITHER(MKS_MINI_12864, ENDER2_STOCKDISPLAY)
   #define _LCD_CONTRAST_MIN  120
   #define _LCD_CONTRAST_INIT 195
-#elif EITHER(MKS_MINI_12864_V3, BTT_MINI_12864_V1)
+#elif ENABLED(FYSETC_MINI_12864_2_1)
   #define _LCD_CONTRAST_MIN  255
   #define _LCD_CONTRAST_INIT 255
```

Comment:
This would seem to be the more correct solution, if all three of these displays are lacking contrast control but need their contrast to be set to 255 during init.

Reply:
If you comment out that `#if` block / those `_LCD_CONTRAST_MIN` & `_LCD_CONTRAST_INIT` lines, you can still adjust the contrast on all three of these displays, but it does look best at max/255 from the three types I have here (Fysetc, MKS, and BTT).

Reply:
What would be the lowest reasonable MIN contrast value?

Reply:
> What would be the lowest reasonable MIN contrast value?

- `MKS_MINI_12864_V3`: `_LCD_CONTRAST_MIN` works until ~180.
- `BTT_MINI_12864_V1`: `_LCD_CONTRAST_MIN` works until ~200, but sometimes the screen goes blank below ~230. It's not consistent. Maybe this one needs to stay at a full `_LCD_CONTRAST_MIN 255` / `_LCD_CONTRAST_INIT 255` to prevent future bug reports.
- `FYSETC_MINI_12864_2_1`: I'm still trying to find the Fysetc version of this LCD, so I'll update this comment once I do.

Reply:
we have tested with fystec and btt modules they were doing good with >201 contrast 
best results aquired with 240 in my case. had a common ws2811.
and yes MKS mini 12864 isn't using any of led driver like other 2 and circuited direct to pins. in this case dip and dim is possible but not with other 2

in case all 3 should work good with 230 pulse

Reply:
The minimum contrast value can be a bit subjective and changes based on the viewing angle.

Ex: We have other LCDs with what I consider low minimum values, but is still technically readable.